ted_sws/core/adapters/logger.py

```diff
@@ -0,0 +1,82 @@
+import abc
+import enum
+import logging
+
+import logstash
+
+from ted_sws import config
+
+
+class LoggingType(enum.Enum):
+    ELK = "ELK"
+    PY = "PY"
+
+
+DOMAIN_LOGGING_TYPES = config.LOGGING_TYPE.split(",") if config.LOGGING_TYPE is not None else [LoggingType.PY.value]
+DOMAIN_LOGGING_TYPE = DOMAIN_LOGGING_TYPES[0]
+
+
+class LoggerABC(abc.ABC):
+    """
+    This abstract class provides methods definitions and infos for available loggers
+    """
+    pass
+
+
+class Logger(LoggerABC):
+    """
+    This class provides common features for available loggers
+    """
+
+    def __init__(self, name: str = __name__):
+        self.name = name
+        self.logger = logging.getLogger(name)
+        self.level = logging.DEBUG
+        self.logger.setLevel(self.level)
+
+    def get_logger(self) -> logging.Logger:
```

Comment:
This looks weird to me. If this is a singleton attempt, then it does not work this way in Python. 
Otherwise, I'm curious about its purpose.

Reply:
The Logger class is a "cover" for specific (PY/ELK) logger. This get_logger method is to retrieve the native Python logger instantiated with this Logger instance. That's all. No singletons, just a getter. :)